source/frontend/src/features/documents/documentUpload/DocumentUploadForm.tsx

```diff
@@ -56,7 +52,7 @@ const DocumentUploadForm: React.FunctionComponent<
   };
 
   const initialFormData = new DocumentUploadFormData(
-    documentStatusTypes[0]?.value?.toString(),
+    props.documentStatusOptions[0]?.value?.toString(),
```

Comment:
nit: would be slightly safer/more readable to search for none instead of hardcoding the first index of the array.